src/fiuba/algo3/tp2/juego/Mapa.java

```diff
@@ -28,6 +29,16 @@ public Casillero getCasillero(Posicion unaPosicion) {
         return this.casilleros.get(fila).get(columna);
     }
 
+    public boolean estaAfueraDelMapaLaPosicion(Posicion unaPosicion) {
+        boolean estaAfuera = false;
+        int fila = unaPosicion.getY();
+        int columna = unaPosicion.getX();
+        if (fila < 0 || columna < 0 ||  fila > (filas - 1) || columna > (columnas - 1)) {
```

Comment:
che no seria this.columnas?

Reply:
Es que no sé, como veo un tablero, la columnas se leen de izquierda a la derecha, entonces representan los x, no?

Reply:
Me refiero que no es smalltalk jaja, accedes a filas con this filas y columnas con this columnas

Reply:
pero no me estoy referiendo a un atributo de mapa, sino creando variables fila y columna para leer la posicion

Reply:
recien entendi. No sé, no me tiró ningun tipo de error asi que supongo que funciona. Lo cambio para homogeneizar.